src/rust/bitbox02/src/ui.rs

```diff
@@ -19,6 +19,8 @@ extern crate alloc;
 use crate::password::Password;
 use alloc::boxed::Box;
 
+const MAX_LABEL_SIZE: usize = bitbox02_sys::MAX_LABEL_SIZE as _;
```

Comment:
Why not `... as usize - 1` to "deduct one for the null terminator" like with `DEVICE_NAME_MAX_LEN`?

Reply:
I mean, I see `MAX_LABEL_SIZE` says it excludes null terminator in the doc comments but `MEMORY_DEVICE_NAME_MAX_LEN` doesn't. Seems inconsistent and easy to miss.

Reply:
Yeah it's inconsistent. Not sure which way is better, without null terminator is better for Rust, but with null terminator is better in C (no need to do `+1` everywhere). Maybe I'll add a 2nd constant for each, `..._WITH_NULLTERMINATOR` :sweat_smile:

Reply:
Maybe just add a comment here why it doesn't have `-1`.

Either way, I think `_WITH_NULL` would be sufficient. No need for long `TERMINATOR` :)

Reply:
Longer term though I hope we'll unify null/no null constants.

Reply:
> Maybe just add a comment here why it doesn't have -1.

Done this for now, as suggested.